src/audiovideocontroller/AudioVideoControllerFacade.ts

```diff
@@ -9,6 +9,14 @@ export default interface AudioVideoControllerFacade {
   addObserver(observer: AudioVideoObserver): void;
   removeObserver(observer: AudioVideoObserver): void;
   start(): void;
+  /**
```

Comment:
Nice explanation, but I think you can move this comment into the `stop()` method in the implementation.

Reply:
Cool, thanks! I have updated it now.